integration/oplog/basic_oplog_test.go

```diff
@@ -0,0 +1,207 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package oplog
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo/options"
+
+	"github.com/FerretDB/FerretDB/integration"
+	"github.com/FerretDB/FerretDB/integration/setup"
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+)
+
+func TestOplogBasic(t *testing.T) {
+	t.Parallel()
+
+	ctx, coll := setup.Setup(t)
+	local := coll.Database().Client().Database("local")
+	ns := fmt.Sprintf("%s.%s", coll.Database().Name(), coll.Name())
+	opts := options.FindOne().SetSort(bson.D{{"$natural", -1}})
+
+	// Create capped collection for oplog if needed
```

Comment:
// cat

Reply:
https://github.com/FerretDB/FerretDB/blob/3db735b9859056dfbd1c8e3fd2c75aa07e896298/CONTRIBUTING.md?plain=1#L263-L270

That's something you still ignore